### ISSUE TITLE
modelscale prediction errors

### ISSUE THREAD

**Issue by Razish** [Razish/japp#387]
on JA+ servers with modelscale set (e.g. jan on KR server), there are prediction errors when standing on their head

**Comment by Razish** [comment]
Ping @Blackwolf1337 did you want to look at this before me :)?

**Comment by EpicLoyd** [comment]
bboxes are not updated?

**Comment by ensiform** [comment]
I don't think they are in JA+ either.

**Comment by Blackwolf1337** [comment]
@Razish I'll take a look at it.

**Comment by Blackwolf1337** [comment]
Ok, I looked over the code and it seems like there is a prediction... the fault is coming from the Ja+ mod itself, since it's not scaling the Boundary Box as @EpicLoyd asked and said. So it is irrelevant to fix it, since it has to do something Serverside. Since the Size Prediction is built in, it bugs for us because the game thinks, that the model is scaled with the boundary box, which isn't. I think this can be closed @Razish ?

**Comment by Razish** [comment]
If we are getting prediction errors, the bounding boxes are out of sync with the server.
Either we are updating them with the modelscale and JA+ server is not, or we are not updating them with the modelscale and JA+ is. Right?

**Comment by EpicLoyd** [comment]
afaik ja+ not updating it because if resulting bbox ( after scaling ) will be too high or too wide that can block whole map or significant part.

**Comment by Blackwolf1337** [comment]
@Razish, that's totally right. But your ja++ Plugin already has a prediction for it, when scaling models up and down you're getting the prediction. It's just Ja+.
